linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/ErrorResponder.scala

```diff
@@ -23,12 +23,18 @@ class ErrorResponder extends SimpleFilter[Request, Response] {
         case ErrorResponder.HttpResponseException(rsp) =>
           rsp
         case _ =>
-          log.error(e, "service failure")
           val message = e.getMessage match {
             case null => e.getClass.getName
             case msg => msg
           }
-          val rsp = Headers.Err.respond(message, Status.BadGateway)
+          val status = e match {
+            case _: TimeoutException | Failure(Some(_: TimeoutException)) =>
+              Status.ServiceUnavailable
```

Comment:
i think, technically, we should be calling getCause on the exception to unwrap it (until getCause returns null).  Finagle does this some places already